types/firestore.d.ts

```diff
@@ -545,6 +545,15 @@ declare namespace FirebaseFirestore {
     close(): Promise<void>;
   }
 
+  /**
+   * An options object that can be used to toggle the default rate limit
```

Comment:
Maybe: `An options object that can be used to disable request throttling in BulkWriter.`

Reply:
much better, done.